### PR TITLE
PS-7793 Document RocksDb variables `rocksdb_alter_column_default_inpl…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ace` and `rocksdb_enable_pipelined_write` (8.0)

 Changes to be committed:
	modified:   source/myrocks/variables.rst

### CODE REVIEW

File: doc/source/myrocks/variables.rst [hunk 1]
```diff
@@ -1354,6 +1373,17 @@ failed insertion attempt in INSERT ON DUPLICATE KEY UPDATE.
...
+
+    :version 8.0.25-15: Implemented
+    :cli: ``--rocksdb-enable-pipelined-write``
+    :dyn: No
+    :scope: Global
+    :vartype: Boolean
+    :default: ``OFF``
+
+DBOptions::enable_pipelined_write for RocksDB.
+
```
Comment (ldonoso) [inline doc/source/myrocks/variables.rst:1371]: My bad, I should have added this information in the ticket.

Might be positive adding some info about the option:

 If enable_pipelined_write is true, separate write thread queue is
 maintained for WAL write and memtable write. A write thread first enter WAL
 writer queue and then memtable writer queue. Pending thread on the WAL
 writer queue thus only have to wait for previous writers to finish their
 WAL writing but not the memtable writing. Enabling the feature may improve
 write throughput and reduce latency of the prepare phase of two-phase
commit.

File: doc/source/myrocks/variables.rst [hunk 2]
```diff
@@ -59,6 +59,10 @@ Also, all variables can exist in one or both of the following scopes:
      - Yes
      - No
      - Global
+   * - :variable:`rocksdb_alter_column_default_inplace`
```
Comment (ldonoso) [inline doc/source/myrocks/variables.rst]: Please, mind that this two variables must go to 5.7. And just `rocksdb_enable_pipelined_write` to 8.0

**Review by ldonoso — CHANGES_REQUESTED** [review]

File: doc/source/myrocks/variables.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst]: Discussed in slack.

File: doc/source/myrocks/variables.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst:1371]: Added.

**Review by ldonoso — APPROVED** [review]

**Pull request merged**